Configuration/DataProcessing/test/RunPromptReco.py

```diff
@@ -29,6 +29,8 @@ def __init__(self):
         self.alcaRecos = None
         self.PhysicsSkims = None
         self.dqmSeq = None
+        self.setRepacked = False
+        self.isRepacked = False
```

Comment:
On 11/10/15 12:04 AM, David Lange wrote:
> In Configuration/DataProcessing/test/RunPromptReco.py
> <https://github.com/cms-sw/cmssw/pull/12326#discussion_r44378627>:
> 
>> @@ -29,6 +29,8 @@ def __init__(self):
>>          self.alcaRecos = None
>>          self.PhysicsSkims = None
>>          self.dqmSeq = None
>> +        self.setRepacked = False
>> +        self.isRepacked = False
> 
> @slava77 <https://github.com/slava77> - I think this would be more
> consistent if the default of isRepacked here is True (as that is default
> elsewhere in the PR)
> 

I'm not sure I understand the logic.

As defined here, the defaults in this file leave it to the corresponding
scenarios to set their values.
The isRepacked has no effect if setRepacked is false.
If the defaults need to be reset (the main purpose of these flags here),
and so far it's only needed in HI, it would be reset to false.


I can change, if you confirm that you still want it at a true value.
(the file is useful only in local tests; in T0 perspective, these flags
are invisible)

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/12326/files#r44378627>.
>